src/injected/ui.cpp

```diff
@@ -6320,40 +6320,100 @@ void load_font()
     ImGuiIO& io = ImGui::GetIO();
     io.FontAllowUserScaling = false;
     PWSTR fontdir;
+
+    ImFontConfig font_config;
+    font_config.MergeMode = true;
+    font_config.EllipsisChar = u'\u2026';
+    static const ImWchar emoji_range[] = {0x1, 0x1FFFF, 0};
+    static const ImWchar ellipsis_range[] = {0x2026, 0x2026, 0}; // one of the asian fonts has a stupid huge ellipsis, we get it explicitly from segoe
+
+    std::string font_default;
+    std::string font_jp;
+    std::string font_ko;
+    std::string font_ru;
+    std::string font_zhcn;
+    std::string font_zhtw;
+    std::string font_emoji;
+
     if (SHGetKnownFolderPath(FOLDERID_Fonts, 0, NULL, &fontdir) == S_OK)
     {
+        using cvt_type = std::codecvt_utf8<wchar_t>;

```

Comment:
Just a tiny thing, you could be using `std::filesystem::path::string::c_str` instead of `std::codecvt` here.

Reply:
On the other hand this code was already here when I got there so I didn't have to write anything

Reply:
I just mentioned it bc it was nicer when stealing this code for PL  :P